pkgs/development/python-modules/orderly-set/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # tests
+  pytestCheckHook,
+  mypy,
+}:
+
+buildPythonPackage rec {
+  pname = "orderly-set";
+  version = "5.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "seperman";
+    repo = "orderly-set";
+    rev = "refs/tags/${version}";
+    hash = "sha256-iQgPTBkyuBqDz64Tws+cqvTgp0UCD8C/5GoTik/mdrE=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [
+    "orderly_set"
+  ];
+  nativeCheckInputs = [
+    pytestCheckHook
+    mypy
```

Comment:
```suggestion
```
For the same reason, there is no need for static analysis.

Reply:
> For the same reason, there is no need for static analysis.

Same with this.

Reply:
> > For the same reason, there is no need for static analysis.
> 
> Same with this.

Found a way - simply disabled the relevant tests..